<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notas</class>
 <widget class="QMainWindow" name="Notas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>573</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Registro de notas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/resource/registros.png</normaloff>:/resource/registros.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 194, 190);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 194, 190);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Estudiantes">
       <attribute name="title">
        <string>Registro de estudiantes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>296</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="cmdListar">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Agregar a la lista.</string>
                </property>
                <property name="text">
                 <string>LISTAR</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="inNombre">
             <property name="toolTip">
              <string>Ingrese nombre y apellido del estudiante.</string>
             </property>
             <property name="placeholderText">
              <string>Ingrese el nombre y apellido del estudiante</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>NOMBRE Y APELLIDO:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>LISTA DE ESTUDIANTES</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPlainTextEdit" name="outLista">
             <property name="toolTip">
              <string>Lista de estudiantes enlistados.</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Notas">
       <attribute name="title">
        <string>Registro de notas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>NOTA 2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="2">
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>70</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="cmdAgregar">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Calcular el promedio total y enlistar.</string>
                </property>
                <property name="text">
                 <string>Promediar</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QDoubleSpinBox" name="inN1">
             <property name="toolTip">
              <string>Ingrese su respectiva nota.</string>
             </property>
             <property name="maximum">
              <double>20.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>ESTUDIANTE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="2">
            <widget class="QDoubleSpinBox" name="inN2">
             <property name="toolTip">
              <string>Ingrese su respectiva nota.</string>
             </property>
             <property name="maximum">
              <double>20.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="inEstudiante">
             <property name="toolTip">
              <string>Seleccione el estudiante.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>NOTA 1</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>PROMEDIOS</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="outNotas">
             <property name="toolTip">
              <string>Lista de estudiantes con sus respectivos promedios.</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>312</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cmdAbierto">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Abrir documento con sus notas.</string>
             </property>
             <property name="text">
              <string>Abrir</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdGuardado">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Guardar documento con sus notas.</string>
             </property>
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="mnuAbrir"/>
    <addaction name="mnuGuardar"/>
    <addaction name="separator"/>
    <addaction name="mnuListar"/>
    <addaction name="separator"/>
    <addaction name="mnuSalir"/>
   </widget>
   <widget class="QMenu" name="menuAy_uda">
    <property name="title">
     <string>Ay&amp;uda</string>
    </property>
    <addaction name="mnuAcerca"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuAy_uda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuAcerca">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/resource/informacion.png</normaloff>:/resource/informacion.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Acerca de</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionA_brir_nuevo">
   <property name="text">
    <string>A&amp;brir </string>
   </property>
  </action>
  <action name="action_Guardar_Archivo">
   <property name="text">
    <string>&amp;Guardar </string>
   </property>
  </action>
  <action name="action_Listar_Etudiantes">
   <property name="text">
    <string>&amp;Listar Etudiantes</string>
   </property>
  </action>
  <action name="actionA_gregar_Notas">
   <property name="text">
    <string>Agregar &amp;Notas</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="mnuAbrir">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/resource/caja.png</normaloff>:/resource/caja.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;brir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mnuGuardar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/resource/salvar.png</normaloff>:/resource/salvar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mnuListar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/resource/listar.png</normaloff>:/resource/listar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Listar Estudiantes</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="mnuAgregar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/resource/añadir.png</normaloff>:/resource/añadir.png</iconset>
   </property>
   <property name="text">
    <string>Agregar &amp;Notas</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="mnuSalir">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/resource/salida.png</normaloff>:/resource/salida.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
